frontend/src/views/Market.vue

```diff
@@ -1149,3 +1341,15 @@ export default Vue.extend({
 }
 
 </style>
+
+function getCharacterName(charId: string): String {
```

Comment:
why are these here?

Reply:
Spaghetti codes. Removed line 1345 and the proceeding codes.